src/pages/RepoPage/CoverageOnboarding/OtherCI/OtherCI.tsx

```diff
@@ -32,9 +34,9 @@ function OtherCI() {
   const uploadToken = orgUploadToken ?? data?.repository?.uploadToken ?? ''
   const tokenCopy = orgUploadToken ? 'global' : 'repository'
 
-  const uploadCommand = `codecovcli upload-process -t ${uploadToken}${
-    orgUploadToken ? `-r ${repo}` : ''
-  }`
+  const uploadCommand = `./codecov -u ${
```

Comment:
Is this suppose to start with `./codecov` or `codecov? Design includes "./" but looking at this comment https://www.figma.com/design/oSgW7AFOXDr1TGrkWh79kV?node-id=1-488#822891221, it is just `codecov`

Reply:
@spalmurray-codecov

Reply:
Yea. I think we agreed to stick with `./codecov`. Reason being the previous step has you download the executable to the current working directory. Without adding codecov to your `$PATH`, it won't work without the `./`. I figure people who care about this difference know what's going on enough to figure out how to do the other way. Important thing is that running the steps explicitly will work.